.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
   global:
     - NODE_ENV=test
     - JWT_SECRET=prfhdfdh
+    - TEST_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NiwidXNlcm5hbWUiOiJncmFjZSIsImVtYWlsIjoiZ3JlZW5AZ3JhY2UuY29tIiwicm9sZSI6InVzZXIiLCJpYXQiOjE1NjI3NDE5MjB9.YAULMmPECVaKm-gDZME8kUmIrPeH6sRpCkGOS987Lf0
```

Comment:
Thank you, for this feedback.
I had to add the jwt token on the `travis.yml` because I noticed that the unit test I created for my feature implementation will require the token when running on Travis

Reply:
Okay. I think it's better generated in your test file on the go rather than making it an environment variable. You can call the jwtSigner function in the utils helper folder to generate it while you pass in a payload as a parameter